### PR TITLE
Add GtkFlowBox

### PR DESCRIPTION
**Pull request by csoriano89**
Hi,
So first time contributing in github and with Rust. So probably I made a mess, but let see =)
I attached a patch with the addition of the GtkFlowBox and another patch only intended for testing for review which should not be committed to master.

Also, it would be awesome to have a way to reach you on IRC or something, I have some questions about the general design.

Cheers

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Hey ! Thanks for your contribution ! You actually can talk on [gitter](https://gitter.im/rust-gnome/gtk). I'm taking a look at your code.

**Comment by GuillaumeGomez** [comment]
All examples are in another repository (rust-gnome/examples to be precise). Except that, your code is good, I have nothing to add. What do you think of it @gkoz ?

**Comment by csoriano89** [comment]
I didn't add all the signals yet, I wanted to have an early preview given my inexperience with Rust and github.

It is necessary to be merged tho right? Or we can add them later. Depending how you normally work.

**Comment by GuillaumeGomez** [comment]
No, I can let this PR open until you say it's ready to be merged. However, I won't merge it until you move the example part into the good repository.

**Comment by gkoz** [comment]
Seems like `gtk::cast` and `gtk_glue.c` bits are missing.
This should be gated on `cfg(feature = "gtk_3_12")` if I'm not mistaken.

**Comment by GuillaumeGomez** [comment]
@gkoz: "GtkFlowBox was added in GTK+ 3.12."

File: src/traits/mod.rs [hunk 1]
```diff
@@ -59,6 +62,7 @@ pub mod misc;
 pub mod button;
 pub mod label;
 pub mod _box;
+pub mod flow_box;
```
Comment (gkoz) [inline src/traits/mod.rs]: This should be preceded by `#[cfg(feature = "gtk_3_12")]`.
And I don't see a similar import in `widgets/mod.rs` at all.

**Comment by GuillaumeGomez** [comment]
[FlowBox](https://github.com/gtk-rs/gtk/blob/master/src/auto/flow_box.rs) type is now generated. I close this PR.

**Pull request closed without merging**